payplug.php

```diff
@@ -100,22 +103,102 @@ private function upgrade()
 					// If is defined
 					else
 						$os_payment = defined('_PS_OS_PAYMENT_');
-					Configuration::updateValue('PAYPLUG_ORDER_STATE_PAID', (int)$os_payment);
+
+					Payplug::updateConfiguration('PAYPLUG_ORDER_STATE_PAID', (int)$os_payment);
 				}
 			}
+			// Add test status && add hook
+			if ($version === false || version_compare($version, '0.9.7', '<'))
```

Comment:
Do you know that PrestaShop can already auto-upgrade the modules ?
http://doc.prestashop.com/display/PS15/Auto-updating+modules